### ISSUE TITLE
Shields breaking text/fonts on some websites

### ISSUE THREAD

**Issue by holdit** [brave/brave-browser#23093]
## Description 
Since version 1.39.111 the text on some websites started to look wrong: wrong font, wrong spacing, wrong line hight, etc. It only happens on some sites. So far I've only noticed it happening on pages without custom fonts (eg: happens on old.reddit.com with the default theme, but not when a custom theme with a different font is used. the new reddit UI is also fine).

This seems to be related to Shield's "Block fingerprinting" option (standard or strict) as disabling it fixes the problem.

## Steps to Reproduce

   1. Enable Shields and set "Block fingerprinting" to "standard" or "strict".
   2. Reboot the browser.
   3. Access a page like: https://old.reddit.com/r/worldnews/

Note: Only seems to happens on pages without custom fonts. A sub (eg: Brave's sub) with a custom theme looks fine (unless you disable custom themes on your reddit account). Please note that this issue is not limited to reddit.

## Actual result:
Wrong spaces, line height, wrong font (number 26):
<img width="283" alt="Screenshot 2022-05-26 at 22 06 56" src="https://user-images.githubusercontent.com/23439751/170583038-c8afeada-6fdf-4702-a698-ce5dc1a88161.png">

Wrong font:
<img width="1496" alt="Screenshot 2022-05-26 at 22 28 50" src="https://user-images.githubusercontent.com/23439751/170583225-86f4d9be-59be-47f8-a246-d43bfc520fa0.png">

## Expected result:
Correct spaces, font, etc:
<img width="223" alt="Screenshot 2022-05-26 at 22 08 46" src="https://user-images.githubusercontent.com/23439751/170583053-aeb483ae-9d49-469c-a3b9-bc0e40c5a349.png">

Using the correct verdana font:
<img width="1496" alt="Screenshot 2022-05-26 at 22 29 14" src="https://user-images.githubusercontent.com/23439751/170583295-2f578c22-0fa3-4ee7-8e42-57ccb320d08d.png">

## Reproduces how often: 
Most of the times the text looks wrong. A browser restart is sometimes needed after enabling Shields' privacy option to see updated results.

## Brave version (brave://version info)
Tested with:

- Stable: Version 1.39.111 Chromium: 102.0.5005.61 (Official Build) (arm64)
- Nightly: Version 1.41.17 Chromium: 102.0.5005.61 (Official Build) nightly (arm64)

<img width="697" alt="Screenshot 2022-05-26 at 22 39 25" src="https://user-images.githubusercontent.com/23439751/170584542-40bc8e85-2d35-4409-abd6-abdabdf8ee60.png">
<img width="697" alt="Screenshot 2022-05-26 at 22 39 13" src="https://user-images.githubusercontent.com/23439751/170584550-b48185bd-10f5-497c-b2fe-f9750f55bce0.png">

Machine:

- MacBook Pro (16-inch, 2021)
- Apple M1 Max
- 32GB RAM
- macOS Monterey 12.4 (stable) and 12.5 (beta) 

According to this comment, it's also happening on Windows 10: https://old.reddit.com/r/brave_browser/comments/uwuad0/release_channel_139111/i9x2ow8/

## Version/Channel Information:
Can reproduce the issue on the current stable and nightly versions.

## Other Additional Information:
No issues when the privacy option of Brave Shields is disabled.

## Miscellaneous Information:
More reports on reddit: https://old.reddit.com/r/brave_browser/comments/uwuad0/release_channel_139111/i9ty88d/

**Comment by ryanbr** [comment]
I'll test Brave/m1 later, but for Windows (w11): @holdit 
![reddit-screeny](https://user-images.githubusercontent.com/1659004/170849883-ea452619-8789-43e3-8666-c18890cd551b.png)

**Comment by holdit** [comment]
@ryanbr I see the same.

Reddit isn't the best example because the site _looks_ fine, but all that is supposed to be using Verdana as font. Instead, it's using Arial(?) the text and Times New Roman(?) in the search bar. If you disable fingerprint blocking, **restart** Brave (very important), and visit the page again, the text will be wider because it's using the proper font.

The place were this issue manifests itself well is on Todoist(.com) dashboard. You'd need to have an account to check it out, but look at the spaces between the fonts on these two screenshots.

If I allow fingerprinting (text looks fine):
<img width="860" alt="Screenshot 2022-05-29 at 05 21 46" src="https://user-images.githubusercontent.com/23439751/170852262-a1414f13-5a79-4bcb-817f-41304c152b46.png">

If I use the standard or strict fingerprinting blocking (spacing is all messed up):
<img width="877" alt="Screenshot 2022-05-29 at 05 21 23" src="https://user-images.githubusercontent.com/23439751/170852395-2ac9b767-ae7b-4377-8dfc-b02f0d039f8d.png">

**Comment by holdit** [comment]
Found another website where the font change also happens: https://www.ispreview.co.uk/

![ezgif com-gif-maker](https://user-images.githubusercontent.com/23439751/170855816-f47b34d4-61f1-4914-8174-445cb527b783.gif)

**Comment by ryanbr** [comment]
Tested in Brave Beta: (win11). Given its holiday in the US, I'll ask internally.

![isp-prev](https://user-images.githubusercontent.com/1659004/170861744-52b3fd06-6020-4787-9c31-398e2cb1f119.png)

**Comment by Latias4Ever** [comment]
I can confirm that this is happening to me as well. Started since the 26th, and the sites where the fonts start changing seem random, one time one particular site is changing fonts, another time it's a different site.

Here's the latest example I have. Shields on:
![image](https://user-images.githubusercontent.com/24550309/170867178-e8c50dbe-3072-49ac-801d-4c943fff788a.png)

Shields off.
![image](https://user-images.githubusercontent.com/24550309/170867184-8a9889a2-7510-4bff-864e-97cdf0c8e96e.png)

**Comment by holdit** [comment]
Another example, NPR's text website: https://text.npr.org/1101971227

Could this be some new fingerprint blocking that was added to the aggressive protection, but is also being used by the regular option? I don't know, maybe something that doesn't reveal the real list of fonts available by the system?

Again, a browser restart after enabling/disabling fingerprint blocking may be required for us to see the differences. Here's a gif showing the difference:

![ezgif com-gif-maker](https://user-images.githubusercontent.com/23439751/170873264-f999f790-c68b-4603-bdf2-8f81c83a9ac2.gif)

**Comment by Cattae** [comment]
+1 Brave Community https://community.brave.com/t/brave-shield-affecting-fonts/398575/8

Other recent font issues which may or may not be related.
https://community.brave.com/t/the-font-courier-new-is-not-showing-any-more/398502
https://community.brave.com/t/weird-blurry-fonts-when-brave-shields-enabled/398392
https://community.brave.com/t/font-not-working-properly-on-latest-v1-39-111-desktop-revision/398074
https://community.brave.com/t/papyrus-font-not-displayed-in-brave-browser/397959
https://community.brave.com/t/brave-browser-displaying-thin-fonts/380058

**Comment by ccampbell** [comment]
Just noticed this in github itself, you can see in the star count section:

With shields on:
<img width="448" alt="image" src="https://user-images.githubusercontent.com/259316/171293841-2627b0aa-b1ca-49bd-b5a1-fbde447e9d14.png">

With shields off:
<img width="468" alt="image" src="https://user-images.githubusercontent.com/259316/171293879-51bc7bac-ff3a-4b0d-a513-d7de0d0be09a.png">

**Comment by holdit** [comment]
Found another one. Feedly.com login page, with tracking protection "standard":

<img width="276" alt="Screenshot 2022-06-01 at 00 51 16" src="https://user-images.githubusercontent.com/23439751/171302118-46f384bb-dc91-492f-b98e-bcfd1b2008b6.png">

Tracking protection "off":

<img width="277" alt="Screenshot 2022-06-01 at 00 51 25" src="https://user-images.githubusercontent.com/23439751/171302132-5ba1b818-55f5-4f95-a5cd-1e6251710ab7.png">

**Comment by ryanccn** [comment]
I have the same issue on macOS arm64 (M1), most notably on GitHub and some other platforms. I had to switch to other browsers because it was making the UI messed up.

For instance, this is what this current page looks like:

<img width="1246" alt="Screen Shot 2022-06-03 at 21 26 36" src="https://user-images.githubusercontent.com/70191398/171863358-b0fb90c9-ffd7-44c5-8b0a-f1c4f6387dff.png">

The web fonts aren't working and have fallbacked to system fonts.

**Comment by ryanccn** [comment]
Notion also doesn't have fonts anymore.

**Comment by gmacar** [comment]
Same bug on macOS arm64 (M1).

**Comment by spiderkitten** [comment]
Yes I am seeing this on Notion.

<img width="1199" alt="Screen Shot 2022-06-07 at 9 58 30 am" src="https://user-images.githubusercontent.com/13283343/172271617-79cb7d9a-fa2b-49c3-932b-70a8aabdc834.png">
 
 It resolves itself when I turn Shields off. 
 
Version 1.39.111 Chromium: 102.0.5005.61 (Official Build) (x86_64)  
Mac OS Mojave 10.14.6

**Comment by gmacar** [comment]
> It resolves itself when I turn Shields off.

yes but it doesn't make sense, because one of Brave's major strengths are Shields and/or the (partial) protection against fingerprinting. If I had to turn them off, I might as well use another browser.

**Comment by TNLuke** [comment]
This is happening on our website on MacOS 11.6.6. Allowing fingerprinting fixes it. We are using a "system font" stack based on [CSS-Tricks](https://css-tricks.com/snippets/css/system-font-stack/) and [normalize.css](https://github.com/necolas/normalize.css):

`font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol","Noto Color Emoji";
`

**Comment by ryanccn** [comment]
Yes @huylv's workaround works for me too

**Comment by holdit** [comment]
Also happening on ghacks: https://github.com/brave/brave-browser/issues/23391

**Comment by bsclifton** [comment]
This was fixed with https://github.com/brave/brave-core/pull/13550 - but we haven't had another 1.39 release

Our next release is going to be product version 1.40.x which is slated for next Tuesday (June 21st). I'll close this issue out as a duplicate of the original issue https://github.com/brave/brave-browser/issues/22605

**Comment by holdit** [comment]
@bsclifton Is this supposed to be working on `1.39.122`?

I'm still seeing the same problem on some sites with Standard + "Prevent sites from fingerprinting me based on my language preferences". For example, the Feedly login page:

<img width="322" alt="Screenshot 2022-06-15 at 14 56 05" src="https://user-images.githubusercontent.com/23439751/173846952-0ce046fb-8b05-4395-b90a-669dd4dcb03b.png">

Same with ispreview.co.uk and other sites/pages I posted above.

I'm seeing the same on `1.40.96` beta.

So, for me at least, the problem isn't fixed.

**Comment by contaoacademy** [comment]
Yes, same problem here on Todoist `V1.39.122`

**Comment by bsclifton** [comment]
Oh wow - ok thanks for the heads up folks - let's re-open this

**Comment by bsclifton** [comment]
@pilgrim-brave was able to confirm issue on Feedly

However the others should be fixed. @rajtslegr and @contaoacademy can you try force-reloading the page? (ex: Ctrl + Shift + F5; or open the dev tools, go to Network tab, tick the `Disable cache` box, and reload page)?

**Comment by TNLuke** [comment]
I'm still seeing the issue on my website. V1.39.122 on MacOS 11.6.6. Force-reloaded and restarted the browser. The workaround of switching off "Prevent sites from fingerprinting me based on my language preferences" resolves the issue.  However, I noticed I'm not seeing the issue on my development site, which is using the same font stack and the same Brave shields.

**Comment by spiderkitten** [comment]
Seems to be fixed on Notion for me, but I am experiencing the different font issue on Reddit. When I change the shield settings (turn it on/off or allow fingerprinting) the font changes. 

Brave V1.39.122 
MacOS 10.14.6

**Comment by holdit** [comment]
ispreview.co.uk looks fine now, but the following are still affected:

todoist.com (with login). Setting off:
<img width="836" alt="Screenshot 2022-06-16 at 00 17 18" src="https://user-images.githubusercontent.com/23439751/173958245-8c6c7595-2dd4-43c5-a8cc-49aeacc260f5.png">

Setting on:
<img width="838" alt="Screenshot 2022-06-16 at 00 17 45" src="https://user-images.githubusercontent.com/23439751/173958275-0971324e-8809-4b1e-abd8-dcbe7e49b9e5.png">


old.reddit.com/r/worldnews/ (no login). Setting off (font: verdana; wider text):
<img width="957" alt="Screenshot 2022-06-16 at 00 19 21" src="https://user-images.githubusercontent.com/23439751/173958347-c3efd830-862a-4ce5-9627-df21b46f7c9e.png">

Setting on (font: arial?):
<img width="802" alt="Screenshot 2022-06-16 at 00 18 37" src="https://user-images.githubusercontent.com/23439751/173958336-d8bcf632-fb4d-4454-9694-8140f6b64f77.png">

text.npr.org. Setting on/off:
![ezgif com-gif-maker](https://user-images.githubusercontent.com/23439751/173958661-c439731a-6bdb-45e2-8d72-1f4830f7cb68.gif)

(Version 1.39.122 Chromium: 102.0.5005.115 (Official Build) (arm64))

**Comment by ersan** [comment]
This is happening to me on my own website (https://valhost.net/demo - toggle "show console"), it's displaying Arial instead of Courier when shields are enabled.  Was driving me nuts trying to figure out what the issue was.

**Comment by contaoacademy** [comment]
> @rajtslegr and @contaoacademy can you try force-reloading the page? (ex: Ctrl + Shift + F5; or open the dev tools, go to Network tab, tick the `Disable cache` box, and reload page)?

I tried, but the problem still exists with Todoist.

**Comment by angelicapuertotello** [comment]
We've received 1 new report about this:

> when the "Block fingerprinting" feature is enabled, fonts in the Todoist web app break.

https://todoist.zendesk.com/agent/tickets/685144

**Comment by angelicapuertotello** [comment]
Two more:

https://todoist.zendesk.com/agent/tickets/685004
https://todoist.zendesk.com/agent/tickets/685110

What should I say to these users? Thanks for looking into it!

**Comment by Nihal247** [comment]
Just to inform, the fonts on ghacks.net are broken as well, and the site is behaving strange, as sometimes the correct fonts load up, but sometimes they don't.

**Comment by angelicapuertotello** [comment]
https://todoist.zendesk.com/agent/tickets/685449

**Comment by holdit** [comment]
Angelica, assuming those tickets are about this issue (we can't see them), you can suggest that users disable the option causing it:

Settings » Shields » Disable "Prevent sites from fingerprinting me based on my language preferences"

Or recommend a different browser (Firefox, Safari, Chrome, etc). Maybe unpopular here, but a browser shouldn't be breaking websites or have terrible performance (#23267). I understand that sometimes features don't work as we expect them to, but to keep it for a month (so far) on what's supposed to be the *stable* channel?

**Comment by angelicapuertotello** [comment]
> Angelica, assuming those tickets are about this issue (we can't see them) 

I added some screenshots to the comments. I didn't know you had no access, sorry.

> you can suggest that users disable the option causing it

I did... to some. Some others already know that's the issue. But they contacted us in hope we can figure out the issue. 

> Or recommend a different browser (Firefox, Safari, Chrome, etc). 

Users know they can switch to a different browser but they asked for our help. That's why I'm trying to see if that's the case.

> but to keep it for a month (so far) on what's supposed to be the _stable_ channel?

I didn't understand this part, sorry.
I raised the issue because we've had an increasing number of users reporting the same thing on macOS. These cases are not older than a week.

Here's another one: https://todoist.zendesk.com/agent/tickets/686237
User: pjbell@gmail.com

> Odd spacing occurring in the past week or more on the website. I'm using Brave browser on MacOS 10.15

https://share.getcloudapp.com/z8umPLNz

If these reports belong somewhere else or you need more information, I'll be happy to help.

**Comment by bsclifton** [comment]
Hi folks - we're working on a fix here. Exploring backing this out versus disabling the feature flag.  Thank you for your patience 😄

**Comment by kjozwiak** [comment]
> Fix will be rolled out soon with our variations server, Griffin. We're testing that now and will close the issue once it's live 😄

Above should be live now 👍 Easiest way to check to see if the feature that was causing the above issues has been disabled via Griffin is going into `brave://wallet` and ensuring that you see the following:

```
DisableReduceLanguage:Default
```

Example:

![image](https://user-images.githubusercontent.com/2602313/175226792-c4e41535-5337-4a27-a1fe-0e7f7ad9669d.png)

**Comment by kjozwiak** [comment]
> > Above should be live now 👍 Easiest way to check to see if the feature that was causing the above issues has been disabled via Griffin is going into `brave://wallet` and ensuring that you see the following:
> 
> You mean brave://version - but yep it's working as expected now - thank you.

yup, definitely meant `brave://version`. Must have copied/pasted the wrong one and didn't proof read it. I updated the above. Glad it's working for you now 👍

**Comment by holdit** [comment]
The problem is back for me with 1.41.96.

Active variations:
AdRewardsStudy:NextPaymentDay
AdServingStudy:DefaultAdNotificationsPerHour=10/MaximumAdNotificationsPerDay=100/MaximumInlineContentAdsPerHour=12/MaximumInlineContentAdsPerDay=60
BraveAds.UserActivityStudy:Triggers=NTPSI/TimeWindow=15m/Threshold=0.0/IdleTimeThreshold=5s
BraveDebounceStudy:Enabled
ConversionsStudy:NewDefaultPattern
Default1pBlockingStudy:Disabled
DisableReduceLanguage:Enabled
EphemeralStorageStudy:Enabled
ExtendedVariationsSafeMode5:SignalAndWriteViaFileUtil5
FirstPartyEphemeralDomainBlockStudy:Enabled
ForceDisableBlinkFeatures:Enabled
MacCoreLocationBackendStudy:Enabled
NativeCosmeticFilteringStudy:Enabled
PartitionBlinkMemoryCacheStudy:Enabled
PartitionConnectionsByNetworkIsolationKeyStudy:Enabled
PermissionLifetimeReleaseStudy:Enabled
RequestAdsEnabledApiStudy:Enabled
SpeedreaderReleaseStudy:Enabled

**Comment by angelicapuertotello** [comment]
@kjozwiak 
From user emilmelnikov@gmail.com: 

> I would like to reopen the ticket: it seems that fonts are broken again in v2290.

Ref: https://todoist.zendesk.com/agent/tickets/685144

**Comment by AshikaDoist** [comment]
I have another user reporting this issue, seems it has returned.

>Interesting.. I switched over to using Brave Browser a while ago and maybe this issue only exists there. I tried on Chrome and it looks much better.

>No idea why it'd look different - every other site I use works/looks the same

![image](https://user-images.githubusercontent.com/60232965/181845721-8bdc6c00-05eb-4c7c-8762-1ac93be35fe6.png)

ref: https://todoist.zendesk.com/agent/tickets/694332

**Comment by jonathanawesome** [comment]
I experienced this issue and arrived here after some basic troubleshooting and finding that turning down shields allowed our font stack to work as expected. At the time, I was running macOS 12.4 and a slightly outdated version of Brave. 

The font stack is `Inter, -apple-system, BlinkMacSystemFont, avenir next, avenir, helvetica neue, helvetica, Ubuntu, roboto, noto, segoe ui, arial, sans-serif`, and the font that I was seeing was `avenir`. So the browser was skipping Inter (despite Inter being installed correctly and having worked as expected with this font stack previously) and also skipping the two apple system font declarations. Again, turning down shields allowed the stack to perform as expected.

Next, I updated Brave to `1.42.86`, which was _after_ the fix announced above, and that did not improve the issue. Only after updating macOS to 12.5 was the issue resolved.

Hope this helps!

**Comment by pilgrim-brave** [comment]
I have opened a followup issue so we can track these 3 new reports (two about Todoist and the latest one from jonathanawesome). Also I am testing a fix locally.

**Comment by AshikaDoist** [comment]
>I’m having a spacing issue using todoist in my brave browser. Attached is a screenshot. Thank you for your assistance.
![image](https://user-images.githubusercontent.com/60232965/183475290-af8b7d84-e0c5-4999-8d17-19ff2bf01c12.png)

ref: https://todoist.zendesk.com/agent/tickets/692482

**Comment by pilgrim-brave** [comment]
AshikaDoist, this issue has been fixed in https://github.com/brave/brave-core/pull/14489, and the fix is being rolled out to all Brave users. Thank you for your report; it was very helpful. And thank you for your patience.

**Comment by dmetzler1988** [comment]
I have the same issue - but with custom fonts (installed locally on my mac (OS 12.5) on the latest Brave Browser (Version 1.42.88 Chromium: 104.0.5112.81 (Official Build) (x86_64)).

If i disable the "Block fingerprint" option it is fine again. With "Default" it uses a wrong font.
On chrome and other browser it works fine.

**Comment by dmetzler1988** [comment]
@bsclifton i currently have the version 1.42.97 - and still same issue. If shields are activated, the fonts are wrong (times new roman i think). If i deactivate the shields, it works as expected. The problem occurs on inbox of outlook.office.com - maybe this will help.

**Comment by bsclifton** [comment]
@dmetzler1988 thanks for confirming - what website are you noticing this on?
cc: @pilgrim-brave

**Comment by dmetzler1988** [comment]
@bsclifton i noticing this on outlook.office.com - on each mail with the custom fonts.
But today i saw something weird which maybe helps - or not xD
I started my Mac, opened my fresh instance of Brave Browser and navigated to the Outlook website. The custom fonts was displayed with active shield.
Later on the day i needed to restart my Mac. So i shut down and start it again. Opened again a fresh instance of Brave Browser, navigated to the Outlook Website and now the custom fonts can't be displayed with active shield. I deactivated shield and reload the page - the custom fonts are displayed. Then i activate the shield again and i see again the fallback font.
Just as information - it seems now, that it works sometimes and sometimes not. The Brave Version is the same than on my post from yesterday.

**Comment by dajare** [comment]
@bsclifton : At the risk of adding a "me too" ... me too! I'm on MacOS 10.14.6 (stuck on Mojave since I need 32 bit libraries) + Brave "Version 1.42.97 Chromium: 104.0.5112.102 (Official Build) (x86_64)". 

I have custom CSS on Wikipedia that uses Merriweather as the main text font. Just recently (in the last week) Brave began not to load Merriweather (on odd occasion it does, unhelpfully). If I turn off "Brave Shield" for https://en.wikipedia.org/, the problem disappears. On that site, I'm happy to leave shields down ... but this ought not to be, as I understand it.

**Comment by AshikaDoist** [comment]
Reports are resurfacing of this happening on Todoist:

>The overall display interface is stripped down- letterspacing is oddly spread out and the date display for each entry is hard to read- characters are truncated on the top and bottom. It's hard to use the app when I'm straining to read it. I've logged out and back in several times and refreshed the page repeatedly. See attached screenshot.
Please advise, looking forward to your reply!

![image](https://user-images.githubusercontent.com/60232965/186277531-a0c622ed-0eb5-4e2f-b89c-cea6b9c57243.png)

User says:
>Hi Ashika- Yes, I am using Todoist on Brave! So strange that the issue just surfaced after I've been using Todoist on Brave for a while now... thanks for the update and I'll check with the Brave folks about their work to address this.

ref: https://todoist.zendesk.com/agent/tickets/698451

**Comment by AshikaDoist** [comment]
Another TD user:
>Also in Brave browser, there are extra spaces between words.

ref: https://todoist.zendesk.com/agent/tickets/698475

**Comment by pilgrim-brave** [comment]
Hi Ashika, thank you for these reports. To the best of my knowledge, this bug was fixed in Brave 1.42.97. Can you find out which version these users are on? Visiting `brave://version` in a new tab will show you the details I need to confirm if this is still happening.

**Comment by dajare** [comment]
> Hi Ashika, thank you for these reports. To the best of my knowledge, this bug was fixed in Brave 1.42.97. 

My entry, three comments above, includes the identification of Brave 1.42.97 as the version which is throwing up problems for me on this front.

**Comment by spiderkitten** [comment]
Apologies if this reopens the issue, but I am still having this problem on Notion. It actually only started in the past couple of days. I originally had this issue all the way back when it was first an issue, but then it was fixed, and now its back again, but this time instead of weird spacing its changing the font to be different/larger. Unfortunately I cannot share a screenshot as its happening on pages I can't share publicly. But, it's happening on the gallery view of databases in notion. 

Brave says it's up to date. 

Brave version 1.42.97 Chromium: 104.0.5112.102 (Official Build) (x86_64) 

Here's a screenshot of the version:  
<img width="645" alt="Screen Shot 2022-08-26 at 8 43 50 pm" src="https://user-images.githubusercontent.com/13283343/186887238-f5ca67d5-025d-4e53-bdf4-d329ec53d2ff.png">

**Comment by YoavRheims** [comment]
Since Version 1.46.144 Chromium: 108.0.5359.128 (Official Build) (64-bit) on Windows 11., I have also the identical issue.

**Comment by kjozwiak** [comment]
@YoavRheims mind providing the websites that you guys are having issues with? @spiderkitten sounds like you had issues with https://todoist.zendesk.com/agent/tickets/698475? Are you still seeing the same problems? It's been sometime since the `1.42.97` release and we've had a few font fixes in between then and now.

**Comment by spiderkitten** [comment]
> @spiderkitten sounds like you had issues with https://todoist.zendesk.com/agent/tickets/698475? Are you still seeing the same problems? It's been sometime since the `1.42.97` release and we've had a few font fixes in between then and now.

I don't know what the todoist link is, I get an error that the page doesnt exisit. But the issue I was having in Notion seems to be all good for me right now, thank you.

**Comment by bsclifton** [comment]
@spiderkitten glad to hear it's OK now 😄 The thing I'd ask folks to check shield settings specifically on the site where they have the problem and ensure you have `Block fingerprinting` (or `Allow fingerprinting`, but that's not preferred) and not `Aggressively block fingerprinting`

**Comment by fischfood** [comment]
I'm not sure if this is the same others have been experiencing, but I noticed a discrepancy on a personal site I use for a simple redirect (basically Jira set to the start of this week + 5 days, with a specific view).

I have the page output "Redirecting to Jira, week of 2023-06-19", but since I'm the only one using it, I didn't bother throwing in a google font call, I just have it set to use a local font for Montserrat.

If I have the google font script in there, it works for both shields up or down.

If I don't have that script and am relying on the local font, shield up blocks it, shields down (or allow fingerprinting) and it loads fine.

I wonder if people still having issues are on a site that has a font-family set that the user has locally, but the site itself does not have the script providing the files

**Comment by stako** [comment]
This issue persists. Locally installed fonts are appearing on pages with fingerprinting **allowed,** however the fonts are not appear on pages with fingerprinting **blocked**

**Comment by kjozwiak** [comment]
> This issue persists. Locally installed fonts are appearing on pages with fingerprinting **allowed,** however the fonts are not appearing on pages with fingerprinting **blocked**
> 
> I see that the previous fix was to add Helvetica to the fonts whitelist. Perhaps Montserrat & Roboto could be added as well.

@stako if you're still having issues with the above, I would suggest creating a new issue using the [template](https://github.com/brave/brave-browser/issues/new?assignees=&labels=OS%2FDesktop&projects=&template=desktop.md&title=). Add as much information as possible. This way it will get looked at and triaged rather than replying to an older issue that has been closed for some time 👍

**Comment by dontdieych** [comment]
> This issue persists. Locally installed fonts are appearing on pages with fingerprinting **allowed,** however the fonts are not appearing on pages with fingerprinting **blocked**
> 
> I see that the previous fix was to add Helvetica to the fonts whitelist. Perhaps Montserrat & Roboto could be added as well.

Same here. Override font using stylus addon does not working with `Block fingerprinting` ON.

```
https://chzzk.naver.com/live/381fc2b3c10c2c438dafe767b48a2ae4

Operating System: Fedora Linux Asahi Remix 40
KDE Plasma Version: 6.1.4
KDE Frameworks Version: 6.5.0
Qt Version: 6.7.2
Kernel Version: 6.10.9-401.asahi.fc40.aarch64+16k (64-bit)
Graphics Platform: Wayland
Processors: 4 × Apple Avalanche (M2), 4 × Apple Blizzard (M2)
Memory: 7.3 GiB of RAM
Graphics Processor: Apple M2
Product Name: Apple MacBook Air (15-inch, M2, 2023)
U-Boot Version: 2024.04
```
